olwebsite-app/src/style-sheets/components/_search-box.scss

```diff
@@ -0,0 +1,31 @@
+.search-box {
+    width: 75%;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    &-icon {
+        color: black;
+        cursor: pointer;
+        padding: 0 0.5rem;
+        align-self: center;
+    }
+
+    &-input-wrapper {
+        width: 100%;
+        height: 2rem;
```

Comment:
What are everyone's thoughts on using rem?
https://medium.com/code-better/css-units-for-font-size-px-em-rem-79f7e592bb97
I've never really used it but always up for learning new css tricks

Reply:
I think this one should be resolved and added to the Coding Standards for CSS. I use it for more conformity, but it can get a little hairy for calculations.

Reply:
i either use pixels or percents usually, but that's because they are more intuitive to me.